plonky2x/core/src/frontend/hash/keccak/mod.rs

```diff
@@ -11,7 +11,8 @@ pub mod keccak256;
 
 impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
     /// WARNING: DO NOT USE IN PRODUCTION, this is unconstrained!
-    pub fn keccak256(&mut self, bytes: &[ByteVariable]) -> Bytes32Variable {
+    pub fn keccak256_witness(&mut self, bytes: &[ByteVariable]) -> Bytes32Variable {
+        // TODO: Need to constrain generator result
```

Comment:
@puma314 Should we constrain this now?

Reply:
no, we don't have the implementation so fine to just leave as long as the name is changed to end with `_witness`